docs/USERGUIDE.md

```diff
@@ -76,6 +76,14 @@ Once you have registered your account, you must configure SASL in your client, s
 
 If your client doesn't support SASL, you can typically use the "server password" (`PASS`) field in your client to log into your account automatically when connecting. Set the server password to `accountname:accountpassword`, where `accountname` is your account name and `accountpassword` is your account password.
 
+# Logging with registered nick
+
+Once you've registered a nick, you can use the following to login:
+```
+/msg NickServ identify <nick> <password>
+```
+Please note that, you unlike other softwares, you cannot first `/nick <nick>` and identify with a password.
```

Comment:
This is the default setup so at this point better to just link [this](https://github.com/ergochat/ergo/blob/master/docs/MANUAL.md#nick-equals-account)

Reply:
Yeah, -1 on mentioning `NS IDENTIFY` here, it won't work at all if you're always-on and in general it's much better to use SASL or PASS-stuffing.

Reply:
Initially I had a thought to explicitly make it appear on the docs since it took it time to figure out the things I mentioned here. But feel free make a decision. I'll leave it from here.

Reply:
Were you not able to get SASL or PASS-stuffing to work in your client?